routes/secure.js

```diff
@@ -0,0 +1,152 @@
+var express   = require('express'),
+    mongoskin = require('mongoskin'),
+    router    = express.Router(),
+    db = mongoskin.db((process.env.MONGOLAB_URI || 'localhost:27017/test'), {safe: true});
+
+    // Thanks to http://webapplog.com/tutorial-node-js-and-mongodb-json-rest-api-server-with-mongoskin-and-express-js/
+    // for the cool help
+
+// Helper Methods
+var wordBank = ['veggie', 'ham', 'bacon', 'marinara', 'mushroom', 'extra', 'cheese', 'pineapple', 'easy-on', 'pepperoni', 'sausage', 'onion', 'lovers', 'supreme'];
```

Comment:
I will absolutely look into making a pizza-key module. First I need to learn how to make a module. *Gulp*.